pages/portfolio.html

```diff
@@ -4,9 +4,37 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="styles/style.css" rel="stylesheet">
+  <title>Portfolio</title>
 </head>
 <body>
-  
+  <center>
+  <h1>Portfolio</h1>
+  <p>
+    While a student in Ada Developer's Academy, I have worked on a variety of projects. Here are some of my favorite ones:
+  </p>
+  <p class="container">
+  <img src="https://imgur.com/a/v49wllZ" width="100" height="100">
+  <img src="https://imgur.com/a/VjZQqKE" width="100" height="100">
```

Comment:
All `img` tags should have an `alt` attribute to make it accessible for screen readers.

Also these are broken images.  The URL goes to a webpage not an image

```suggestion
  <img src="https://i.imgur.com/hNVeoXt.jpeg" width="100" height="100" alt="VCR">
  <img src="https://i.imgur.com/beWdHXf.jpeg" width="100" height="100" alt="Todo list">
```